addons/l10n_in_withholding/models/account_move.py

```diff
@@ -197,53 +210,58 @@ def _l10n_in_is_warning_applicable(self, section_id):
             case _:
                 return False
 
+    def _l10n_in_is_tds_tcs_applicable(self, section_alert):
+        self.ensure_one()
+        existing_section = (self.l10n_in_withhold_move_ids.line_ids | self.line_ids).tax_ids.l10n_in_section_id
+        return section_alert not in existing_section
+
     def _get_l10n_in_tds_tcs_applicable_sections(self):
         def _group_by_section_alert(invoice_lines):
             group_by_lines = {}
             for line in invoice_lines:
-                group_key = line.account_id.l10n_in_tds_tcs_section_id
+                group_key = line.account_id.l10n_in_tds_tcs_section_id.id
```

Comment:
```suggestion
                group_key = line.l10n_in_tds_tcs_section_id.id
```
Can we do it with `_read_group()` ? just a thought and suggestion if yes then this method would not be needed anymore 🤔